File: plugin/src/main/groovy/com/sensorsdata/analytics/android/plugin/SensorsAnalyticsJSRAdapter.java
/*
 * Created by renqingyou on 2018/12/01.
 * Copyright 2015－2022 Sensors Data Inc.
 *
 * Licensed under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License.
 * You may obtain a copy of the License at
 *
 *       http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software
 * distributed under the License is distributed on an "AS IS" BASIS,
 * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 * See the License for the specific language governing permissions and
 * limitations under the License.
 */
package com.sensorsdata.analytics.android.plugin;

import org.objectweb.asm.MethodVisitor;
import org.objectweb.asm.commons.JSRInlinerAdapter;

public class SensorsAnalyticsJSRAdapter extends JSRInlinerAdapter {
    protected SensorsAnalyticsJSRAdapter(int api, MethodVisitor mv, int access, String name, String desc, String signature, String[] exceptions) {
        super(api, mv, access, name, desc, signature, exceptions);
    }
}

File: sa-gradle-plugin/src/main/kotlin/com/sensorsdata/analytics/android/plugin/visitor/SensorsAnalyticsJSRAdapter.kt
package com.sensorsdata.analytics.android.plugin.visitor

import org.objectweb.asm.MethodVisitor
import org.objectweb.asm.commons.JSRInlinerAdapter

class SensorsAnalyticsJSRAdapter constructor(
    api: Int,
    mv: MethodVisitor?,
    access: Int,
    name: String?,
    desc: String?,
    signature: String?,
    exceptions: Array<String>?
) : JSRInlinerAdapter(api, mv, access, name, desc, signature, exceptions)